dd-sdk-android-compose/build.gradle.kts

```diff
@@ -100,6 +101,11 @@ unMock {
 }
 
 kotlinConfig()
+taskConfig<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    kotlinOptions {
+        freeCompilerArgs += listOf("-Xopt-in=kotlin.RequiresOptIn")
```

Comment:
So in order to use this API you need to specifically `optIn` in the compiler args ?

Reply:
Yes https://kotlinlang.org/docs/opt-in-requirements.html. My reasoning behind using that is that once `compose` package is released, its version won't reflect that it is still experimental (because it will be common version, as for other packages), so it is an additional precaution to warn the users.

Reply:
Hmm...in that case shouldn't you explain it in the docs better ?

Reply:
Not sure, IDE gives you a warning about that and says what you actually need to do.